Bai8/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bai8
+{
+    class Program
+    {
+
+        public static void miniMaxSum(List<int> arr)
+        {
+            long sum1 = 0, sum2 = 0;
+            int min = int.MaxValue;
+            int max = int.MinValue;
+            List<int> maxSum = new List<int>();
+            List<int> minSum = new List<int>();
+            for (int i = 0; i < arr.Count ; i++)
+            {
+                maxSum.Add(arr[i]);
+                minSum.Add(arr[i]);
+                min = Math.Min(arr[i], min);
+                max = Math.Max(arr[i], max);
```

Comment:
```suggestion
                min = Math.Min(arr[i], min);
                max = Math.Max(arr[i], max);
```
chỗ này sao k dùng linq lấy ra max, min của array luôn ở ngoài vòng for